src/main/java/org/prebid/server/bidder/HttpBidderRequester.java

```diff
@@ -351,6 +348,13 @@ void addHttpCall(BidderCall<T> bidderCall, Result<List<BidderBid>> bidsResult) {
             if (bidderErrors != null) {
                 errorsRecorded.addAll(bidderErrors);
             }
+
+            final List<FledgeAuctionConfig> fledgeAuctionConfigs = bidsResult != null
+                    ? bidsResult.getFledgeAuctionConfigs()
+                    : null;
+            if (fledgeAuctionConfigs != null) {
+                fledgeRecorded.addAll(fledgeAuctionConfigs);
+            }
```

Comment:
Cover with tests, please.

Reply:
Coverage report says this is covered; 162 of 167 lines covered and the missing lines are not from this PR.

Reply:
<img width="1034" alt="Screenshot 2023-01-19 at 13 05 48" src="https://user-images.githubusercontent.com/45672370/213426785-9085b4c7-e1a2-4722-ba5d-87346162e6a2.png">

Reply:
I don't think so :)

Reply:
In intelliJ IDEA
<img width="335" alt="Screenshot 2023-01-19 at 13 10 42" src="https://user-images.githubusercontent.com/56919177/213427558-3ae375df-0bfc-46c1-adb8-dc8373348d06.png">

Reply:
I see, I usually run all the tests and it showed green but it turns out it was an indirect hit from another test.
Thank you for keeping me honest, I'll add some coverage here.